rust/kernel/virtio.rs

```diff
@@ -0,0 +1,187 @@
+// SPDX-License-Identifier: GPL-2.0
+
+//! Virtio.
+//!
+//! C header: [`include/linux/virtio.h`](../../../../include/linux/virtio.h)
+
+use crate::{
+    bindings, device, driver, error::from_kernel_result, str::CStr, to_result,
+    types::PointerWrapper, Result, ThisModule,
+};
+
+/// A registration of a platform driver.
+pub type Registration<T> = driver::Registration<Adapter<T>>;
+
+/// Id of an Virtio device.
+#[derive(Clone, Copy)]
+pub struct DeviceId {
+    /// Device id.
+    pub device: u32,
+
+    /// Mask that identifies which bits are valid in the device id.
+    pub vendor: u32,
```

Comment:
Should this be called `vendor_mask`?

Reply:
Or is the comment incorrect?

Reply:
This comment is incorrect. This should be "Vendor id of the virtio device". I forget to change it after copying from other places.